tests/bootstrap.php

```diff
@@ -5,27 +5,27 @@
  * @package Core_Sitemaps
  */
 
-$_tests_dir = getenv( 'WP_TESTS_DIR' );
+$core_sitemaps_tests_dir = getenv( 'WP_TESTS_DIR' );
 
-if ( ! $_tests_dir ) {
-	$_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
+if ( ! $core_sitemaps_tests_dir ) {
+	$core_sitemaps_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
 }
 
-if ( ! file_exists( $_tests_dir . '/includes/functions.php' ) ) {
-	echo "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL; // WPCS: XSS ok.
+if ( ! file_exists( $core_sitemaps_tests_dir . '/includes/functions.php' ) ) {
+	echo "Could not find $core_sitemaps_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL; // WPCS: XSS ok.
 	exit( 1 );
 }
 
 // Give access to tests_add_filter() function.
-require_once $_tests_dir . '/includes/functions.php';
+require_once $core_sitemaps_tests_dir . '/includes/functions.php';
 
 /**
  * Manually load the plugin being tested.
  */
-function _manually_load_plugin() {
```

Comment:
I'm not sure that any of these changes in this file are necessary, but if you're changing this function name, you'll also need to change it in the `tests_add_filter()` callback below.

Reply:
@joemcgill These were changed because they are what were failing the sniffs:

<img width="640" alt="Screenshot 2019-10-18 at 14 45 34" src="https://user-images.githubusercontent.com/182015/67099658-4aa24080-f1b6-11e9-849d-ea3e7c52e077.png">

Should we be changing the rule in `phpcs.xml.dist` instead to get around this?

Reply:
Oh interesting. Yeah, that makes sense to me, in case this got installed with another plugin declaring the same symbols.